storage/innobase/xtrabackup/test/t/pxb-2710-predict_memory.sh

```diff
@@ -6,6 +6,7 @@ require_debug_pxb_version
 MYSQLD_EXTRA_MY_CNF_OPTS="
 innodb-flush-log-at-trx-commit=0
 sync-binlog=0
+innodb-redo-log-capacity=512M
```

Comment:
if you run the same test against 8.0.29 server, this might fail

Reply:
Fixed by Marcelo in 8.0.29 branch